### PR TITLE
Add bare metal getting started guide

### PR DESCRIPTION
**Pull request by abemusic**

### CODE REVIEW

**Review by yakbulut — COMMENTED** [review]
LGTM

File: docs/current/bare-metal.md [hunk 1]
```diff
@@ -17,6 +17,280 @@
...
+docker pull thecsikos/vaquero:latest
+```
+
+## Prepare vaquero source of truth (SoT) directory
+
+For our purposes, we'll be using the local directory mode for the vaquero server's SoT. You can do the same using a github repo, however you will need to make sure your server is routable to the internet and available to receive webhooks which is outside the scope of this getting started guide.
+
+### Initial directory structure
+
+We'll use /var/vaquero-demo, but feel free to use a different location. Take care to keep things consistent:
```
Comment (fattarsi) [inline docs/current/bare-metal.md]: I had a little trouble parsing this. Maybe consider, 
```In this example our SoT is in /var/vaquero-demo, if you use a different location be sure to update the following commands accordingly.```

**Review by fattarsi — APPROVED** [review]
LGTM

File: docs/current/bare-metal.md (hunk 1 above)
Comment (abemusic) [reply docs/current/bare-metal.md]: You're right....how would you feel if I just took it out. I think it's implied that if you start typing other paths YMMV :)

**Review by fattarsi — APPROVED** [review]
LGTM

**Review by jmeridth — APPROVED** [review]

**Pull request merged**